auth/service/user.go

```diff
@@ -59,27 +52,18 @@ func (d *AuthService) CreateUser(ctx context.Context, user *auth.User) (id int64
 		time.Now(),
 		time.Now(),
 		"system",
-	)
+	).Scan(&insertId)
```

Comment:
```suggestion
	).Scan(&id)
```

ini pake named return, bisa manfaatin itu